requirements.txt

```diff
@@ -5,6 +5,7 @@ graviton@git+https://github.com/algorand/graviton@v0.8.0
 mypy==0.991
 pytest==7.2.0
 pytest-cov==3.0.0
+pytest-custom-exit-code==0.3.0
```

Comment:
Allows succeeding a pytest run with no actual tests run if supply the argument `--suppress-no-test-exit-code`